include/drivers/spi.h

```diff
@@ -129,11 +129,16 @@ extern "C" {
  *    to act as a CS line
  * @param delay is a delay in microseconds to wait before starting the
  *    transmission and before releasing the CS line
+ * @param gpio_dt_flags is the devicetree flags corresponding to how the CS
+ *    line should be driven. Currently ACTIVE_LOW / ACTIVE_HIGH is ignored
```

Comment:
It's not "currently", it's the way it should always be. Because CS may not be gpio driven and thus SPI_CS_ACTIVE_* are the mandatory ones to follow in this particular case.